src/main/java/no/nav/veilarbdirigent/client/veilarbaktivitet/VeilarbaktivitetClientImpl.java

```diff
@@ -69,4 +60,22 @@ public Try<String> lagAktivitet(String data, UUID oppfolgingsPeriodeId) {
             return Try.failure(e);
         }
     }
+
+    static final String KAFKA_CONSUMER_FEATURE_TOGGLE_NAME = "veilarbdirigent.oppfolgingsperiode.consumer.disabled";
+
+    @Override
+    public Try<Boolean> getKafkaFeatureToggle() {
+        String url = UrlUtils.joinPaths(apiUrl, format("/veilarbaktivitet/api/feature?feature=%s", KAFKA_CONSUMER_FEATURE_TOGGLE_NAME));
+        Request request = new Request.Builder()
+                .url(url)
+                .get()
+                .addHeader(AUTHORIZATION, createBearerToken(serviceTokenSupplier.get()))
+                .build();
+        try(Response response = client.newCall(request).execute()) {
+            Map<String, Boolean> map = JsonUtils.fromJson(response.body().string(), Map.class);
```

Comment:
Ikke så sikker på om den deserialiseringen vil funke. Jackson er litt odd når det gjelder Collection types og Maps.
Tror kanskje man må bruke noe a.la
`com.fasterxml.jackson.databind.type.TypeFactory#constructMapType(java.lang.Class<? extends java.util.Map> mapclass, java.lang.Class<?> keyclass, java.lang.Class<?> valueclass) `
for å få typereferanse og så kalle 
```
 public static <T> T fromJson(InputStream inputStream, TypeReference<T> type) {
        return objectMapper.readValue(inputStream, type);
    }
```
Det er utilityfunksjoner i JsonUtils for lister og arrays, men ikke for maps.